bom/runtime/pom.xml

```diff
@@ -273,6 +273,18 @@
                 <artifactId>quarkus-development-mode-spi</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-ide-launcher</artifactId>
+                <version>${project.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <!-- This has shaded dependencies -->
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
```

Comment:
Have you checked it works properly? We had a couple of bad surprises with wildcard exclusions.

Reply:
It appears to, none of the bootstrap deps ended up in the final application.